includes/InputToEntity/FullWikibaseItemInput.php

```diff
@@ -20,10 +22,13 @@ public function __construct( Deserializer $deserializer ) {
 	}
 
 	public function getItem( array $inputEntity ) : Item {
-		if ( !array_key_exists( 'type', $inputEntity ) || $inputEntity['type'] !== 'item' ) {
-			die( 'Only supported entity type is \'item\'' );
+		$supportedTypes = [ 'item' ];
+		if ( !array_key_exists( 'type', $inputEntity ) || !in_array( $inputEntity['type'], $supportedTypes ) ) {
+			throw new ReconciliationException(
+				MessageValue::new( 'wikibasereconcileedit-fullwikibaseiteminput-unsupported-type' )
+					->textListParams( $supportedTypes )
```

Comment:
`->numParams( count( $supportedTypes ) )`

Reply:
Why is the number of supported types relevant??

Reply:
It’s used in the message to decide whether it should say “type” or “types”.

Reply:
Oh, ok. :)